src/server/req_delete.c

```diff
@@ -335,8 +335,8 @@ static void
 decr_single_subjob_usage(job *parent)
 {
 	parent->ji_qs.ji_svrflags &= ~JOB_SVFLG_ArrayJob; /* small hack to decrement usage for a single un-instantiated subjob */
-	account_entity_limit_usages(parent, NULL, NULL, DECR); /* for server limit */
-	account_entity_limit_usages(parent, parent->ji_qhdr, NULL, DECR); /* for queue limit */
+	account_entity_limit_usages(parent, NULL, NULL, DECR, ETLIM_ACC_ALL); /* for server limit */
+	account_entity_limit_usages(parent, parent->ji_qhdr, NULL, DECR, ETLIM_ACC_ALL); /* for queue limit */
```

Comment:
Here we have to `DECR`all the 4 limit attributes. Hence passing `ETLIM_ACC_ALL`

Reply:
As mentioned in the function header comment, this is invoked only for un-instantiated queued subjobs which never ran, hence no need to check for ji_etlimit_decr_queued  flag